content/_data/changelogs/lts.yml

```diff
@@ -9733,6 +9733,526 @@
+    message: |-
+      Replace browser confirm with modal dialogs at many places.
+      Add API for alert, confirm, prompt, modal and form dialogs.
+  - type: major rfe
+    category: rfe
+    pull: 8418
+    authors:
+    - mawinter69
+    - janfaracik
+    - NotMyFault
+    pr_title: "Replace hetero-list YUI button and menu with new style button and tippy.js menu"
+    references:
+      - pull: 8381
+      - pull: 8376
+      - pull: 8375
+      - pull: 8363
+      - pull: 8180
+    message: |-
+      Updates to various UI elements.
+      Modernize buttons, menus, link design, and content blocks.
+  - type: major rfe
+    category: rfe
+    pull: 8403
+    authors:
+      - timja
+    pr_title: "Add Appearance system configuration page"
+    message: |-
+      Add <strong>Appearance</strong> system configuration page to customize Jenkins' look and feel.
+  - type: major rfe
+    category: rfe
+    pull: 8528
+    authors:
+    - jglick
+    - Vlatombe
+    pr_title: "Avoid saving disabled status when deleting a project"
+    references:
+      - pull: 8494
+      - pull: 8395
+      - pull: 8299
+    message: |-
+      Various performance optimizations.
+      Optimizations for loading, label parsing, and project deletion.
+  - type: major bug
+    category: bug
+    pull: 8448
+    authors:
+      - Vlatombe
+    pr_title: "Label parsing robustness fix"
+    message: |-
+      Prevent incorrect <code>readResolve</code> implementations from breaking agent label parsing.
+  - type: rfe
+    category: rfe
+    pull: 8526
+    authors:
+      - basil
+    pr_title: "Automate Java version recommendation administrative monitor"
+    message: |-
+      Automate the display of an administrative monitor when approaching Java end of life (EOL) dates.
+  - type: rfe
+    category: rfe
+    pull: 8250
+    issue: 71087
+    authors:
+      - daniel-beck
+    pr_title: "[JENKINS-71087] Nicer 404 error pages"
+    message: |-
+      Add a nicer 404 error page.
+  - type: rfe
+    category: rfe
+    pull: 8503
+    authors:
+      - basil
+    pr_title: "Remove KXML2 library"
+    message: |-
+      Stop shipping <code>net.sf.kxml:kxml2</code> because Jenkins no longer depends on it.
+  - type: rfe
+    category: rfe
+    pull: 8503
+    references:
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.8
+        title: Remoting 4.8 changelog
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.9
+        title: Remoting 4.9 changelog
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.10
+        title: Remoting 4.10 changelog
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.11
+        title: Remoting 4.11 changelog
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.12
+        title: Remoting 4.12 changelog
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.13
+        title: Remoting 4.13 changelog
```

Comment:
Nice use of the references, but since we're dropping support for those older versions, I don't think it helps to include links to their changelogs.  We haven't linked to the unsupported versions in the previous removal.
```suggestion
```